### PR TITLE
Login

### PR DESCRIPTION
**Pull request by ag-mahoon**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,23 +1,134 @@
...
+app.post("/searchuser",(req, res) => {
+    let userExist = 0;
+    let foundUser ;
+    const users = readData();
+    const { name } = req.body;
+    let allFindUsers = [];
+    
+    for (let i = 0; i < users.length; i ++) {
+
+        if ( name === users[i].name) {
```
Comment (ag-mahoon) [inline server.js]: ager email se search krna chai tu?

Comment (ammarmahoon) [reply server.js]: uski b condition add kr daity

Comment (ag-mahoon) [reply server.js]: haan kr do aur aik conditon ye bhi add kr do k agr user half email address de ya half name de tbhi search results show hon

Comment (ammarmahoon) [reply server.js]: ye half wali condition phly waly k sth hi kesy attach hogi bhla??

**Pull request merged**